src/clj/vanilla/url_handlers.clj

```diff
@@ -25,40 +25,124 @@
     (log/info "Tried to subscribe a nil user to " services)))
 
 
-(def from-sqlite {:id :key :data_source :data-source :data_grid :data-grid})
-(def to-sqlite {:key :id :data-source :data_source :data-grid :data_grid})
 
 
-; fixup the key changes caused by SQLite with rename-keys
-; :id -> :key
+;;;;;;;;;;
+;; Layouts
+
+
+; fixup the key changes caused by SQL with rename-keys
+; :id -> key
 ; :data_source -> :data-source
 ; :data_grid -> :data-grid
+(def from-sql {:id :key :data_source :data-source :data_grid :data-grid})
+(def to-sql {:key :id :data-source :data_source :data-grid :data_grid})
+
 (defn get-layout
   "Retrieves a users layout from the db"
   [user]
-  (map #(clojure.set/rename-keys % from-sqlite) (db/get-user-layout db/vanilla-db {:username user})))
+  (map #(clojure.set/rename-keys % from-sql) (db/get-user-layout db/vanilla-db {:username user})))
 
 
 (defn save-layout
-  "Saves a users entire widget layout to the db"
+  "Saves a users entire widget layout to the db.
+  Take in the new layout, and transformed the data into a workable format for the
+  postgres database."
+  ;@TODO - Clean up the let block? It is working and all steps are necessary - see comment block below
   [new-layout]
-  ;(prn "SAVE-LAYOUT Handler: " new-layout)
+  (prn "SAVE-LAYOUT Handler(not ordered): " new-layout)
```

Comment:
re-comment prn, cleanup the log/info's below.  Maybe just one log/info?  or none?